src/skimage/_build_utils/version.py

```diff
@@ -1,49 +1,56 @@
 #!/usr/bin/env python3
-"""Extract version number from __init__.py"""
 
-import os
+"""Determine and print version number.
 
+Used in top level ``meson.build``.
+"""
 
-def git_version(version):
-    """Append last commit date and hash to version, if available"""
-    import subprocess
-    import os.path
+import subprocess
+from pathlib import Path
 
-    git_hash = ''
+
+def version_in_init():
+    """Extract version string from ``skimage/__init__.py``."""
+    skimage_init = Path(__file__).parent / '../__init__.py'
+    assert skimage_init.is_file()
+
+    with skimage_init.open("r") as file:
+        data = file.readlines()
+
+    version_line = next(line for line in data if line.startswith('__version__ ='))
+    version = version_line.strip().split(' = ')[1].replace('"', '').replace("'", '')
+    return version
+
+
+def try_append_git_commit_n_date(version):
```

Comment:
A more intuitive function name would be helpful here.

Maybe `append_git_revision_and_date`?